src/main/resources/db/migration/V1_6__remove_question_constraints.sql

```diff
@@ -0,0 +1,4 @@
+
+ ALTER TABLE question DROP CONSTRAINT question_code_uniq;
```

Comment:
since the constraint was already removed in `prod`, this won't deploy. i would suggest dropping it manually in `dev` for testing and putting it back in manually.

constraints have to be put in manually anyway (since k8s kills pods before they can be applied)

Reply:
Yes that's fine

Reply:
@ushkarev I've remove the file to remove!